src/main/java/com/example/onlinebookstore/dto/cart/AddToCartRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.onlinebookstore.dto.cart;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class AddToCartRequestDto {
+    @NotNull
+    private Long bookId;
+    @NotNull
+    private int quantity;
```

Comment:
In this case, user can specify a negative number.

Reply:
```suggestion
    @Positive
    private int quantity;
```